settings.gradle

```diff
@@ -1 +1 @@
-rootProject.name = 'example'
+rootProject.name = 'max-skill-trim'
```

Comment:
This was bugging me that these were still `'example'`, because Intellij was still referencing the project like that, so I just updated them haha. Hopefully that's no problem!